src/actions/guides/database/models.cr

```diff
@@ -367,11 +367,11 @@ class Guides::Database::Models < GuideAction
 
     The `Comment` model now has a `commentable` method which could return a `photo` object
     or a `video` object depending on which was associated.
-    <!-- go go polymorphin power rangers -->
+    <!-- go go polymorphic power rangers -->
```

Comment:
😂  hahahah! I actually spelled it with `morphin` on purpose because they were "mighty morphin power rangers".  It was fun, but we can just remove this line now 🤣

Reply:
Ah, I was suspecting there was something to it but couldn't figure it out. Reverted this one.